docs/architecture/state.md

```diff
@@ -1,15 +1,22 @@
-# State
-The state of the Miden rollup describes the current condition of all accounts and note states. It describes what is currently the case. With its state model, using concurrent off-chain state, Polygon Miden aims to realise private, and parallel transaction execution and state bloat minimization. Midens aims to realise:
+Miden rollup state describes the current condition of all accounts and notes; i.e. what is currently the case. 
 
-* **Notes and nullifiers** ensure privacy of note consumption
-* **Flexible data storage** for users who can store their data off-chain or with the network
-* **Parallel transactions** executed concurrently by distinct actors
-* **Concurrent state model** allows block production without knowing the full state
+As the state model uses concurrent off-chain state, Polygon Miden aims for private, and parallel transaction execution and state bloat minimization. 
```

Comment:
nit: here we use `off-chain` and sometimes (architecture overview) we use `offchain` can you quickly make those consistent?